protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/toconnectschema/ProtobufSchemaToConnectSchemaConverter.java

```diff
@@ -125,6 +127,9 @@ private SchemaBuilder toConnectSchemaBuilderForField(final Descriptors.FieldDesc
                 if (fieldDescriptor.getMessageType().getFullName().equals("google.type.TimeOfDay")) {
                     schemaBuilder = Time.builder();
                 }
+                if (fieldDescriptor.getMessageType().getFullName().equals("additionalTypes.Decimal")) {
+                    schemaBuilder = Decimal.builder(DECIMAL_DEFAULT_SCALE);
```

Comment:
Shouldn't this scale be interpreted from Protobuf schema?

Reply:
We need the actual data to be populated to know the scale, we don't know the scale yet from the schema alone.

Reply:
Should it be defined as metadata? Can this scale be changed when we parse the schema?

Reply:
According to the util method fromDecimalProto you defined in the design doc, it pulls the scale from the decimalproto and sets that during the conversion. Is that what you are referring to?